src/firefox/remote.js

```diff
@@ -4,15 +4,53 @@ import {RemoteTempInstallNotSupported, WebExtError} from '../errors';
 import defaultFirefoxConnector from 'node-firefox-connect';
 
 const log = createLogger(__filename);
+
+
+// Flow Types
+
+import type FirefoxClient from 'firefox-client';
+
+export type FirefoxConnectorFn =
+  (port?: number) => Promise<FirefoxClient>;
+
+export type ConnectExtraParams = {
+  connectToFirefox: FirefoxConnectorFn,
+};
+
+export type FirefoxRDPAddonActor = {
+  id: string,
+  actor: string,
+};
+
+export type FirefoxRDPResponseError = {
+  error: {
+    message: string,
+  },
+};
+
+export type FirefoxRDPResponseAny = Object;
+
+export type FirefoxRDPResponseAddon = {
+  addon: FirefoxRDPAddonActor,
+};
+
+export type FirefoxRDPResponseMaybe =
+  FirefoxRDPResponseError | FirefoxRDPResponseAddon | FirefoxRDPResponseAny;
```

Comment:
... that really small bit of haskell that I learned in the past is taking over from inside me :-)

I tried to find a better workaround than creating a type alias of Object, unfortunately this reduces the safety of the type check a bit (a good bit actually :-( ), but at least leave a trace of our intentions and it is a documentation of what the method should return. (basically it is a bit more useful for humans, it makes not a big difference for to the machines ;-)).